poetry.lock

```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
Is your local version of poetry `>=1.3`? In the CI and for building docker images, the poetry version is set at `1.2.2`, until we see necessary to upgrade this, please downgrade locally and rerun the poetry lock command to regenerate the lock file. Sorry to be a pain 😅 I'm only concerned that having a lock file generated by a different poetry version could cause issues with future releases.

Reply:
No worries, will fix. Haven't payed much attention to poetry version, but totally makes sense.